homeassistant/components/zha/core/group.py

```diff
@@ -150,9 +150,7 @@ def endpoint(self) -> zigpy.endpoint.Endpoint:
     def members(self) -> list[ZHAGroupMember]:
         """Return the ZHA devices that are members of this group."""
         return [
-            ZHAGroupMember(
-                self, self._zha_gateway.devices.get(member_ieee), endpoint_id
-            )
+            ZHAGroupMember(self, self._zha_gateway.devices[member_ieee], endpoint_id)
```

Comment:
There is a check in the list comprehension for `if member_ieee in self._zha_gateway.devices` so we know it is present.